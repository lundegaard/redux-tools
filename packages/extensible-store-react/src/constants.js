export const SUFFIX_DELIMITER = '__id:';
