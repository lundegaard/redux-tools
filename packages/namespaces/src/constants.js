export const DEFAULT_FEATURE = 'namespaces';
