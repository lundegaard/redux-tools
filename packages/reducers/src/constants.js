export const ROOT_KEY = '@redux-tools/reducers/ROOT_KEY';
