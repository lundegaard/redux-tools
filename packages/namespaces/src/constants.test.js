import * as Constants from './constants';

describe('constants', () => {
	it('should not be changed', () => {
		expect(Constants).toMatchSnapshot();
	});
});
