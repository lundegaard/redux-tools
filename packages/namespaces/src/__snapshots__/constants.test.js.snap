// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should not be changed 1`] = `
Object {
  "DEFAULT_FEATURE": "namespaces",
  "__esModule": true,
}
`;
